application/config.json.template

```diff
@@ -110,5 +110,7 @@
         "special": [
         ]
     },
+    "memberCountCategoryName": "Info",
```

Comment:
should be a pattern not just a string, change suffix to Pattern and code to use a Pattern.compile(...) and then pattern.matches(...) similiar to other entries in our config

Reply:
not done